spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -195,9 +195,11 @@
     end
   end
 
-  context 'with doc_auth_selfie_capture_enabled set to true' do
+  context 'with idv_allow_selfie_check_in_login_prod? set to true' do
```

Comment:
This is a persnickety nit.  It feels odd to state that a method which returns a boolean is set to true.  Maybe try a rephrase like the below?
```suggestion
  context 'with idv_allow_selfie_check_in_login_prod? returning true' do
```

Reply:
Good point. Updated.